### ISSUE TITLE
libavformat branch: Audio with OF 0.10?

### ISSUE THREAD

**Issue by michaelbaisch** [bangnoise/ofxHapPlayer#43]
Hello,

I've tried using the same Hap file with audio which worked with 0.9.8 on macOS now with 0.10.0 and it doesn't work. When loading the movie the console says:

```
kAudioUnitErr_TooManyFramesToProcess : inFramesToProcess=1024, mMaxFramesPerSlice=128
```

When trying to play the movie the console says:

```
[warning] ofRtAudioSoundStream: stream over/underflow detected
```

Has anybody got audio to work with OF 0.10.0?

Greetings.

**Comment by michaelbaisch** [comment]
After doing a bit more debugging with 0.9.8 I realized that I also get both of those console outputs there, while audio is working. So I'm not sure what the difference is.

**Comment by squaresquid** [comment]
Same isue on windows 10 with OF 0.10. 
Testet same HAP files with 0.9.8 and sounds works fine.

**Comment by bangnoise** [comment]
Hi - apologies for the delay on this. Fixed in 90fdc23. The ofSoundStream setup method we were using got broken in OF 10.0, I've updated to avoid the breakage.

Re-open this issue if you still have problems with sound.
